test_scripts/API/SetAudioStreamingIndicator/ideas_SetAudioStreamingIndicator_conditions_for_SDL_to_handle_related_cases.txt

```diff
@@ -0,0 +1,111 @@
+./test_scripts/API/SetAudioStreamingIndicator/001_ATF_N_TC_SetAudioStreamingIndicator_PLAY_PAUSE_IGNORED_media_app.lua Check_that_SDL_sends_IGNORED_media_app_already_set_AudioStreamingIndicator_PLAY_PAUSE
+./test_scripts/API/SetAudioStreamingIndicator/002_ATF_N_TC_SetAudioStreamingIndicator_PLAY_IGNORED_media_app.lua Check_that_SDL_sends_IGNORED_media_app_already_set_AudioStreamingIndicator_PLAY
+./test_scripts/API/SetAudioStreamingIndicator/003_ATF_N_TC_SetAudioStreamingIndicator_PAUSE_IGNORED_media_app.lua Check_that_SDL_sends_IGNORED_media_app_already_set_AudioStreamingIndicator_PAUSE
+./test_scripts/API/SetAudioStreamingIndicator/004_ATF_N_TC_SetAudioStreamingIndicator_PLAY_PAUSE_IGNORED_media_app_fake_param.lua Check_that_SDL_sends_IGNORED_media_app_already_set_AudioStreamingIndicator_mobile_request_fake_param_from_other_RPC_PLAY_PAUSE
+./test_scripts/API/SetAudioStreamingIndicator/005_ATF_N_TC_SetAudioStreamingIndicator_PLAY_IGNORED_media_app_fake_param.lua Check_that_SDL_sends_IGNORED_media_app_already_set_AudioStreamingIndicator_mobile_request_fake_param_from_other_RPC_PLAY
+./test_scripts/API/SetAudioStreamingIndicator/006_ATF_N_TC_SetAudioStreamingIndicator_PAUSE_IGNORED_media_app_fake_param.lua Check_that_SDL_sends_IGNORED_media_app_already_set_AudioStreamingIndicator_mobile_request_fake_param_from_other_RPC_PAUSE
+./test_scripts/API/SetAudioStreamingIndicator/007_ATF_N_TC_SetAudioStreamingIndicator_non-media_app_REJECTED.lua Check_that_SDL_sends_REJECTED_to_non-media_app_for_RPC_SetAudioStreamingIndicator
+./test_scripts/API/SetAudioStreamingIndicator/008_ATF_N_TC_SetAudioStreamingIndicator_navi_app_REJECTED.lua Check_that_SDL_sends_REJECTED_to_navi_app_for_RPC_SetAudioStreamingIndicator
```

Comment:
Why do we need separate case for navi- and comm- apps?
The question is applicable for 008-009, 011-012, 019-024, 027-028, 061-062, 065-066, 074-081, 084-085